invenio_app_rdm/communities_ui/views/communities.py

```diff
@@ -106,6 +109,21 @@ def communities_home(pid_value, community, community_ui):
         )
 
 
+@pass_community(serialize=True)
+def communities_browse(pid_value, community, community_ui):
+    """Community browse page."""
+    permissions = community.has_permissions_to(MEMBERS_PERMISSIONS)
```

Comment:
What are the required permission for the browse page? I am lacking context on why `MEMBERS_PERMISSIONS` is used here.

Reply:
It's to avoid `jinja2.exceptions.UndefinedError: 'permissions' is undefined`. I'm not sure what to pass instead?